analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UriShouldNotBeHardcoded.cs

```diff
@@ -74,3 +73,20 @@ void ValidCases(string s)
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/7815
+class ReproFN_7815
+{
+    class MyClass
+    {
+        public string FilePath { get; set; }
+    }
+
+    void Method()
+    {
+        var myClass = new MyClass
+        {
+            FilePath = "/my/other/folder" // FN
```

Comment:
On line 45 the same example says (Compliant - we ignore unix paths by default). So I am not sure this is a good reproducer for a FN.